src/solver/simulation/common-hydro-remix.cpp

```diff
@@ -220,6 +221,92 @@ static bool Remix(const Data::AreaList& areas,
     return status;
 }
 
+std::vector<double> computeTotalGenWithoutHydro(const std::vector<double>& load,
+                                                const std::vector<double>& unsupE,
+                                                const std::vector<double>& hydroGen)
+{
+    // Can be computed (for any hour) as : load - unsupplied energy - hydro
+    std::vector<double> to_return = load;
+    for (size_t i = 0; i < to_return.size(); ++i)
+    {
+        to_return[i] -= unsupE[i] + hydroGen[i];
+    }
+    return to_return;
+}
+
+std::vector<double> extractLoadForCurrentWeek(const Data::Area& area,
+                                              const unsigned int year,
+                                              const unsigned int firstHourOfWeek)
+{
+    std::vector<double> load_to_return(168, 0.);
+    for (int h = 0; h < 168; h++)
```

Comment:
Don't use 168, use a constant instead

Reply:
ok, **168** was also used in previous hydro remix algo, as well as in many places in Simulator.
I'll locally change this, but we should think about doing something about it